docs/content/mobile/android/local-deploy.md

```diff
@@ -31,6 +34,9 @@ You need the Unity Remote app for this. See the [Unity documentation](https://do
 1. In the **Editor Settings** window's **Unity Remote** section, set the **Device** setting to **Any Android Device**.
 1. Open the Scene that starts both your [client-workers]({{urlRoot}}/content/glossary#client-worker) and [server-workers]({{urlRoot}}/content/glossary#server-worker).
 1. On your mobile device, open the **Unity Remote** app. Make sure you allow it permissions for location and camera.
+1. If your project does not already contain one, create a MonoBehaviour script which inherits from the [`MobileWorkerConnector`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs) and include the functionality you want. <br> 
```

Comment:
as these steps are the same for all three of them: should we move it up to "Prepare your project to connect to a local deployment" ?

Reply:
Agree with this. Adding the information under the "Prepare your project to connect to a local deployment" section makes the page cleaner and the steps easier to read.

Adding the following under Prepare:

Create and add a MonoBehaviour script to your Android client-worker GameObject.

1. If your project does not already contain one, create a MonoBehaviour script which inherits from the [`MobileWorkerConnector`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs) and include the functionality you want.
1. In the Unity Editor, add the MonoBehaviour script to your Android client-worker GameObject. In the FPS Starter Project, navigate to **Assets** > **Fps** > **Prefabs** > **AndroidClientWorker**.
1. In the Unity Editor, navigate to your Android client-worker GameObject and ensure the `ShouldConnectLocally` checkbox is checked in the script's drop-down window of the Inspector window.